<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>348</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LoginWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widgetLogin" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>430</width>
       <height>330</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>430</width>
       <height>330</height>
      </size>
     </property>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>431</width>
        <height>331</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageMain">
       <widget class="QWidget" name="widgetLoginTop" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>430</width>
          <height>130</height>
         </rect>
        </property>
        <widget class="QToolButton" name="toolButtonClose">
         <property name="geometry">
          <rect>
           <x>398</x>
           <y>1</y>
           <width>32</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/res/login/close.png</normaloff>:/res/login/close.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QToolButton" name="toolButtonSetting">
         <property name="geometry">
          <rect>
           <x>334</x>
           <y>0</y>
           <width>32</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/res/login/setting.png</normaloff>:/res/login/setting.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QToolButton" name="toolButtonMin">
         <property name="geometry">
          <rect>
           <x>366</x>
           <y>0</y>
           <width>32</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/res/login/min.png</normaloff>:/res/login/min.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="labelBgMovie">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>430</width>
           <height>130</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <zorder>labelBgMovie</zorder>
        <zorder>toolButtonClose</zorder>
        <zorder>toolButtonMin</zorder>
        <zorder>toolButtonSetting</zorder>
       </widget>
       <widget class="QWidget" name="widgetLoginBottom" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>130</y>
          <width>430</width>
          <height>200</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <widget class="QPushButton" name="pushButtonLogin">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>150</y>
           <width>250</width>
           <height>32</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>登录</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBoxAutoLogin">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>120</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>自动登录</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBoxRememberPwd">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>120</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>记住密码</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonRegister">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>170</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>注册账号</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QToolButton" name="toolButtonFindPwd">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>120</y>
           <width>60</width>
           <height>18</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>找回密码</string>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditPwd">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>90</y>
           <width>250</width>
           <height>24</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>密码</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditOuterInput">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>48</y>
           <width>250</width>
           <height>24</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>请输入账号</string>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="placeholderText">
          <string>MQ号码</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonDropDown">
         <property name="geometry">
          <rect>
           <x>316</x>
           <y>48</y>
           <width>24</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
       <widget class="HeadShadowWidget" name="widgetHeadShadow" native="true">
        <property name="geometry">
         <rect>
          <x>175</x>
          <y>90</y>
          <width>80</width>
          <height>80</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:40px;
border:1px solid white;</string>
        </property>
        <widget class="QLabel" name="labelHeadImage">
         <property name="geometry">
          <rect>
           <x>8</x>
           <y>8</y>
           <width>64</width>
           <height>64</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius:31px;
border:1px solid white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="pageLogin">
       <widget class="QToolButton" name="toolButtonSettingLogin">
        <property name="geometry">
         <rect>
          <x>334</x>
          <y>0</y>
          <width>32</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/login/setting.png</normaloff>:/res/login/setting.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QToolButton" name="toolButtonCloseLogin">
        <property name="geometry">
         <rect>
          <x>398</x>
          <y>0</y>
          <width>32</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/login/close.png</normaloff>:/res/login/close.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QToolButton" name="toolButtonMinLogin">
        <property name="geometry">
         <rect>
          <x>366</x>
          <y>0</y>
          <width>32</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/login/min.png</normaloff>:/res/login/min.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="labelBgMovieLogin">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>430</width>
          <height>330</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="HeadShadowWidget" name="widgetHeadShadowLogin" native="true">
        <property name="geometry">
         <rect>
          <x>175</x>
          <y>95</y>
          <width>80</width>
          <height>80</height>
         </rect>
        </property>
        <widget class="QLabel" name="labelHeadImageLogin">
         <property name="geometry">
          <rect>
           <x>8</x>
           <y>8</y>
           <width>64</width>
           <height>64</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButtonLoginCancel">
        <property name="geometry">
         <rect>
          <x>165</x>
          <y>250</y>
          <width>100</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
       <zorder>labelBgMovieLogin</zorder>
       <zorder>toolButtonSettingLogin</zorder>
       <zorder>toolButtonMinLogin</zorder>
       <zorder>toolButtonCloseLogin</zorder>
       <zorder>widgetHeadShadowLogin</zorder>
       <zorder>pushButtonLoginCancel</zorder>
      </widget>
      <widget class="QWidget" name="pageLoginError">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widgetLoginErrorTop" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <widget class="QToolButton" name="toolButtonCloseLoginError">
           <property name="geometry">
            <rect>
             <x>398</x>
             <y>0</y>
             <width>32</width>
             <height>32</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/res/login/close.png</normaloff>:/res/login/close.png</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QToolButton" name="toolButtonMinLoginError">
           <property name="geometry">
            <rect>
             <x>366</x>
             <y>0</y>
             <width>32</width>
             <height>32</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/res/login/min.png</normaloff>:/res/login/min.png</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QToolButton" name="toolButtonSettingLoginError">
           <property name="geometry">
            <rect>
             <x>334</x>
             <y>0</y>
             <width>32</width>
             <height>32</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/res/login/setting.png</normaloff>:/res/login/setting.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widgetLoginError" native="true">
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>30</y>
             <width>40</width>
             <height>40</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="res.qrc">:/res/login/waring-fill.png</pixmap>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>30</y>
             <width>291</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>你输入的账户名或密码不正确，你要找回密码嘛？</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>70</y>
             <width>221</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>如果你的密码丢失或遗忘，可访问MQ中心</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonFindPwd2">
           <property name="geometry">
            <rect>
             <x>320</x>
             <y>68</y>
             <width>61</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>找回密码</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widgetLoginErrorBottom" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
          <widget class="QPushButton" name="pushButtonFindPwd">
           <property name="geometry">
            <rect>
             <x>255</x>
             <y>4</y>
             <width>80</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>找回密码</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonCancelLogin">
           <property name="geometry">
            <rect>
             <x>340</x>
             <y>4</y>
             <width>75</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageSettings"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>HeadShadowWidget</class>
   <extends>QWidget</extends>
   <header>feature_widgets/HeadShadowWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
